src/utils/validation.js

```diff
@@ -43,4 +46,19 @@ const validateFileExists = (filePath, errorText) => {
   }
 };
 
-module.exports = { throwError, validate, validateFileExists, validateElement, validateSection };
+const validateFileAccess = (filePath, accessLevel) => {
+  const ACCESS_LEVELS = {
+    all: fs.constants.F_OK,
```

Comment:
shouldn't all be: fs.constants.R_OK | fs.constants.W_OK, fs.constants.X_OK.

I guess this only means the files exists.

Reply:
Maybe this can be renamed to default and then add an all for all access permissions?

Reply:
According to [this website](https://riptutorial.com/node-js/example/5637/check-permissions-of-a-file-or-directory)
`fs.constants.F_OK` means that it has read/write/execute permissions.
Technically, it's the same as `fs.constants.R_OK | fs.constants.W_OK | fs.constants.X_OK` but it's safer to use just `F_OK` if you're on windows machine